pyasic/config/__init__.py

```diff
@@ -286,14 +286,23 @@ def from_iceriver(cls, web_userpanel: dict) -> "MinerConfig":
 
     @classmethod
     def from_luxos(
-        cls, rpc_tempctrl: dict, rpc_fans: dict, rpc_pools: dict, rpc_groups: dict
+        cls,
+        rpc_tempctrl: dict,
+        rpc_fans: dict,
+        rpc_pools: dict,
+        rpc_groups: dict,
+        rpc_config: dict,
+        rpc_profiles: list[dict],
```

Comment:
This type hint is wrong, should just be `dict`.

Reply:
Fixed in [1aeb10e](https://github.com/UpstreamData/pyasic/pull/262/commits/1aeb10e3535e3775ef4a932c04349a977dcc5d85)